frontend/package.json

```
@@ -18,6 +18,7 @@
     "test": "jest",
     "debug-test": "node --inspect-brk node_modules/.bin/jest --runInBand",
     "webdriver-update": "webdriver-manager update --gecko=false",
+    "webdriver-update:osx": "CHROME_VERSION=$(/Applications/Google\\ Chrome.app/Contents/MacOS/Google\\ Chrome --version) && yarn webdriver-update --versions.chrome=\"${CHROME_VERSION}\"",
```

nit: I'd avoid the `:` in the name. maybe

```suggestion
    "webdriver-update-mac": "CHROME_VERSION=$(/Applications/Google\\ Chrome.app/Contents/MacOS/Google\\ Chrome --version) && yarn webdriver-update --versions.chrome=\"${CHROME_VERSION}\"",
```

---

I was going to comment about that, we lack a naming convention for "sibling" tasks.  I generally like the `:` for things that are siblings, but I'll update to a `-`.